build.sbt

```diff
@@ -767,6 +768,23 @@ lazy val circe = (projectMatrix in file("json/circe"))
   .jsPlatform(scalaVersions = List(scala2_12, scala2_13) ++ scala3, settings = commonJsSettings)
   .dependsOn(core, jsonCommon)
 
+lazy val jsoniter = (projectMatrix in file("json/jsoniter"))
```

Comment:
You should add jsoniter to `allAggregates` variable and probably also to `docs` as dependency